discovery-infra/test_infra/helper_classes/kube_helpers/infraenv.py

```diff
@@ -255,17 +255,17 @@ def deploy_default_installenv(
         label_selector: Optional[Dict[str, str]] = None,
         ignition_config_override: Optional[str] = None,
         **kwargs,
-) -> InstallEnv:
+) -> InfraEnv:
 
-    install_env = InstallEnv(kube_api_client, name)
+    install_env = InfraEnv(kube_api_client, name)
     try:
         if 'filepath' in kwargs:
-            _create_installenv_from_yaml_file(
+            _create_infraenv_from_yaml_file(
                 install_env=install_env,
```

Comment:
I think we want to change the variable name too

Reply:
Done